src/components/ui/image/image-spacer.tsx

```diff
@@ -7,11 +7,11 @@ const ImageContainer = styled.div`
   grid-column: 1 / 4;
 
   display: grid;
-  height: 210px;
+  height: 240px;
   margin-top: 80px;
 
   ${up('md')} {
-    height: 360px;
+    height: 620px;
     margin-top: 180px;
   }
 `;
```

Comment:
I think the image spacer it to high in some formats (screenshot is from a 13" MacBook) but I am not a designer 😅 I think it makes sense to ask the design team if a third breakpoint with a third height (between 240 and 620) for the image spacer would be helpful

<img width="1552" alt="Bildschirm­foto 2023-03-02 um 17 29 05" src="https://user-images.githubusercontent.com/57712895/222491047-3e11c67a-0889-4275-8b57-7965bed74cd2.png">